symfony/console/3.3/bin/console

```diff
@@ -15,15 +14,11 @@ if (!class_exists(Application::class)) {
     throw new \RuntimeException('You need to add "symfony/framework-bundle" as a Composer dependency.');
 }
 
-if (!isset($_SERVER['APP_ENV'])) {
-    if (!class_exists(Dotenv::class)) {
-        throw new \RuntimeException('APP_ENV environment variable is not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file.');
-    }
-    (new Dotenv())->load(__DIR__.'/../.env');
-}
-
 $input = new ArgvInput();
 $env = $input->getParameterOption(['--env', '-e'], $_SERVER['APP_ENV'] ?? 'dev', true);
```

Comment:
This means that the environment from `.env` will not be considered anymore as fallback

Reply:
I think i got you now, i think we should default this to null if `$_SERVER['APP_ENV']` does not exist. That way the .env file will still be loaded and that APP_ENV will be used.